pilbox/app.py

```diff
@@ -138,7 +138,7 @@ def __init__(self, **kwargs):
         tornado.web.Application.__init__(self, self.get_handlers(), **settings)
 
     def get_handlers(self):
-        return [(r"/", ImageHandler)]
+        return [(r"/(.*)", ImageHandler)]
```

Comment:
The other changes you made just appear to be refactors. This particular change is the one that satisfies the ticket requirements, correct?